_posts/2017-07-11-Editathon.markdown

```diff
@@ -0,0 +1,155 @@
+---
+layout: epic
+title: Art + Feminism and Artsy Wikipedia/Wikidata Editathon
+date: 2017-07-06
+categories: [Technology, emission, reaction, react-native, react, javascript]
+author: orta
+---
+
+Artsy has always had a focus on Arts meet Science, and we [hosted a meet-up last weekend][meetup-msg] that really hits on both. We had a collection of Artsy Staff, members of [Art + Feminism][afem] NYC, the CocoaPods Peer Lab and volunteers from Wikimedia all helping.
```

Comment:
Worth linking to http://artsy.github.io/blog/2015/08/10/peer-lab/ or http://peerlab.community maybe?